### PR TITLE
44 display outfits

### LINKED ISSUE #44: Display outfits
Display outfits on outfit page
Allow users to make all outfits public at once

### PR DESCRIPTION
**Pull request by uityh**
Implemented displaying outfits in outfits page. Making all outfits public at once. Deleting individual outfits.

### CODE REVIEW

File: routes/outfits.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const express = require("express");
 const router = express.Router();
-
+const outfitValidation = require("../validation/outfit_validation");
 const gen_outfitData = require("../data/gen_outfit");
 const outfitsData = require("../data/outfits");
 const clothesData = require("../data/clothes");
```
Comment (LDomadia) [inline routes/outfits.js:6]: Lines 7 - 10 can be removed since they aren't being used in this file

**Review by LDomadia — COMMENTED** [review]
I'm missing the outfit_validation.js file when running the branch after a git pull. Does the file need to be committed and pushed?

File: routes/outfits.js (hunk 1 above)
Comment (uityh) [reply routes/outfits.js:6]: Just pushed the file and removed those lines.

**Review by LDomadia — APPROVED** [review]
Just a typo in the modal
Screen Shot 2022-05-06 at 1 51 57 PM

**Pull request merged**